sdk/identity/Azure.Identity.Broker/tests/DefaultAzureCredentialFactoryTests.cs

```diff
@@ -57,7 +58,7 @@ public void ValidateDefaultAzureCredentialAZURE_TOKEN_CREDENTIALS_Honored_WithBr
                     Assert.IsTrue(chain.Any(cred => cred is VisualStudioCredential));
                     Assert.IsTrue(chain.Any(cred => cred is AzureDeveloperCliCredential));
                     Assert.IsTrue(chain.Any(cred => cred.GetType() == typeof(VisualStudioCodeCredential)));
```

Comment:
Nit: While you're in here, update this line of code for consistency.

```suggestion
                    Assert.IsTrue(chain. Any(cred => cred is VisualStudioCodeCredential));
```